app/views/events/_event.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="card" id="<%= dom_id event %>">
+  <div class="card-header">
+    <div class="row">
+      <div class="col-md-9">
+        <%= link_to event.identifier_name, event %>
+      </div>
+      <div class="col-md-3">
```

Comment:
Not sure we need the team name here... or if we do, that column split needs to be different.
![image](https://github.com/GatherPack/gatherpack/assets/834099/cdc87588-a10d-4d6e-9575-5aad1bc56ff7)

Reply:
Also if you're keeping this you should throw the team type icon in there too.